docs/top-level-api.md

```diff
@@ -66,6 +66,12 @@ Navigation.startTabBasedApp({
       screen: 'example.SecondSideMenu', // unique ID registered with Navigation.registerScreen
       passProps: {} // simple serializable object that will pass as props to all top screens (optional)
     },
+    style: {
```

Comment:
We should mention that these styles (the entire `style` object actually) are for iOS only